src/frontend/src/common/components/library/data_subview/index.tsx

```diff
@@ -134,14 +134,15 @@ const DataSubview: FunctionComponent<Props> = ({
 }: Props) => {
   // we are modifying state using hooks, so we need a reducer
   const [state, dispatch] = useReducer(searchReducer, {
-    results: data,
+    results: Object.values(data ?? {}),
     searching: false,
   });
 
-  const [checkedSamples, setCheckedSamples] = useState<string[]>([]);
+  const [dataValues, setDataValues] = useState<Sample[] | Tree[]>();
```

Comment:
I was thinking that if data was particularly large, it might be inefficient to be constantly calling Object.values(data) over and over again, and better to use state as a way of not having to recalculate it.